Library/Formula/dcmtk.rb

```diff
@@ -12,19 +17,22 @@ class Dcmtk < Formula
 
   option "with-docs", "Install development libraries/headers and HTML docs"
   option "with-openssl", "Configure DCMTK with support for OpenSSL"
+  option "with-libiconv", "Build with brewed libiconv. Dcmtk and system libiconv can have problems with utf8."
 
   depends_on "cmake" => :build
   depends_on "doxygen" => :build if build.with? "docs"
   depends_on "libpng"
   depends_on "libtiff"
   depends_on "openssl"
+  depends_on "homebrew/dupes/libiconv" => :optional
 
   def install
     ENV.m64 if MacOS.prefer_64_bit?
 
     args = std_cmake_args
     args << "-DDCMTK_WITH_DOXYGEN=YES" if build.with? "docs"
-    args << "-DDCMTK_WITH_OPENSSL=YES" if build.with? "openssl"
+    args << "-DDCMTK_WITH_OPENSSL=YES -DWITH_OPENSSLINC=#{Formula["openssl"].opt_prefix}" if build.with? "openssl"
```

Comment:
My bad. I had that in for the slightly older version/PR but it does look good with this version.